src/app/detail/[task_id]/[user_type]/page.tsx

```diff
@@ -71,6 +78,8 @@ export default async function TaskDetailPage({
   const task = await getOneTask(token, task_id)
   const assignee = addTypeToAssignee(await getAssigneeList(token))
   const attachments = await getAttachments(token, task_id)
+  const activities = await getActivities(token, task_id)
```

Comment:
I believe these three can be done in parallel using Promise.all

Reply:
Solved on c9243f2. ref : https://github.com/copilot-platforms/tasks-app/commit/c9243f2b925e3f122d7abe481c5a038f6639944c